packages/nimble-components/src/menu/menu.foundation.ts

```diff
@@ -0,0 +1,403 @@
+import { DOM, observable } from '@microsoft/fast-element';
```

Comment:
nit: may as well add a comment linking to the original source in this file too like you did in the tests.